### PR TITLE
update reset model

### PR DESCRIPTION
**Pull request by ouyangyu**
使用 batchnorm_relu 以及 batchnorm_add_relu，更新ResNet50

### CODE REVIEW

**Review by ShawnXuan — COMMENTED** [review]
Please append test result in PR.

**Comment by ouyangyu** [comment]
``` 
validation: epoch 47, iter 125, top_1: 0.760460, top_k: 0.928760, samples/s: 2173.505 1599761642.6572137
train: epoch 48, iter 100, loss: 1.872034, top_1: 0.777363, top_k: 0.921133, samples/s: 2897.681 1599761677.6515865
train: epoch 48, iter 200, loss: 1.907133, top_1: 0.776045, top_k: 0.921221, samples/s: 2946.417 1599761712.405689
train: epoch 48, iter 300, loss: 1.881571, top_1: 0.777900, top_k: 0.920986, samples/s: 2960.226 1599761746.9976268
train: epoch 48, iter 400, loss: 1.865137, top_1: 0.779326, top_k: 0.921797, samples/s: 2943.345 1599761781.787998
train: epoch 48, iter 500, loss: 1.998808, top_1: 0.776924, top_k: 0.920420, samples/s: 2956.155 1599761816.4275517
train: epoch 48, iter 600, loss: 1.810058, top_1: 0.777793, top_k: 0.921172, samples/s: 2954.126 1599761851.0909574
train: epoch 48, iter 700, loss: 1.843636, top_1: 0.780342, top_k: 0.922588, samples/s: 2947.964 1599761885.8268173
train: epoch 48, iter 800, loss: 1.885278, top_1: 0.778271, top_k: 0.922539, samples/s: 2945.621 1599761920.590261
train: epoch 48, iter 900, loss: 1.943432, top_1: 0.779355, top_k: 0.922813, samples/s: 2953.276 1599761955.2636473
train: epoch 48, iter 1000, loss: 1.928975, top_1: 0.777607, top_k: 0.921299, samples/s: 2953.482 1599761989.9345825
train: epoch 48, iter 1100, loss: 1.936143, top_1: 0.778486, top_k: 0.922363, samples/s: 2948.229 1599762024.6672919
train: epoch 48, iter 1200, loss: 1.962628, top_1: 0.778076, top_k: 0.922705, samples/s: 2944.443 1599762059.4446678
Saving model to ./output/snapshots/model_save-20200910200345/snapshot_epoch_48.
validation: epoch 48, iter 125, top_1: 0.761240, top_k: 0.928920, samples/s: 2134.385 1599762100.1017818
train: epoch 49, iter 100, loss: 1.849224, top_1: 0.779316, top_k: 0.922129, samples/s: 2896.231 1599762135.1592135
train: epoch 49, iter 200, loss: 1.993535, top_1: 0.776934, top_k: 0.921230, samples/s: 2952.492 1599762169.8417583
train: epoch 49, iter 300, loss: 1.891371, top_1: 0.778145, top_k: 0.921660, samples/s: 2946.291 1599762204.5973327
train: epoch 49, iter 400, loss: 1.937300, top_1: 0.778682, top_k: 0.921641, samples/s: 2950.551 1599762239.302696
train: epoch 49, iter 500, loss: 1.983842, top_1: 0.776650, top_k: 0.919541, samples/s: 2947.592 1599762274.0429325
train: epoch 49, iter 600, loss: 1.869871, top_1: 0.778730, top_k: 0.921533, samples/s: 2948.816 1599762308.7695808
train: epoch 49, iter 700, loss: 1.880350, top_1: 0.777852, top_k: 0.922588, samples/s: 2954.276 1599762343.4304013
train: epoch 49, iter 800, loss: 1.912684, top_1: 0.779736, top_k: 0.922168, samples/s: 2942.320 1599762378.233675
train: epoch 49, iter 900, loss: 1.931287, top_1: 0.778662, top_k: 0.922031, samples/s: 2959.806 1599762412.8297048
train: epoch 49, iter 1000, loss: 1.804406, top_1: 0.778740, top_k: 0.922422, samples/s: 2949.373 1599762447.549054
train: epoch 49, iter 1100, loss: 1.883346, top_1: 0.778770, top_k: 0.922334, samples/s: 2950.431 1599762482.2557244
train: epoch 49, iter 1200, loss: 1.902986, top_1: 0.778223, top_k: 0.922295, samples/s: 2947.210 1599762517.0004823
Saving model to ./output/snapshots/model_save-20200910200345/snapshot_epoch_49.
validation: epoch 49, iter 125, top_1: 0.760020, top_k: 0.929020, samples/s: 2119.166 1599762557.8487167
```

**Comment by ouyangyu** [comment]
- 训练参数
```
python3 of_cnn_train_val.py \
     --train_data_dir=$DATA_ROOT/train \
     --train_data_part_num=256 \
     --val_data_dir=$DATA_ROOT/validation \
     --val_data_part_num=256 \
     --num_nodes=1 \
     --gpu_num_per_node=8 \
     --optimizer="sgd" \
     --momentum=0.875 \
     --label_smoothing=0.1 \
     --learning_rate=1.024 \
     --loss_print_every_n_iter=100 \
     --batch_size_per_device=128 \
     --val_batch_size_per_device=50 \
     --num_epoch=50 \
     --model="resnet50" 2>&1 | tee ${LOGFILE}
```

**Review by ShawnXuan — APPROVED** [review]
